doc/conda/jenkins-conda.rst

```diff
@@ -22,7 +22,15 @@ Also to find the configuration repo, look for the :meth:`get_config_pkg` in the
 .. literalinclude:: meta.yaml.template
     :language: yaml+jinja
 
-Locally test the recipe by running the following command ``docker run -e {config_repo}=/path/to/config/repo -v /path/to/your/repos:/home/saluser/develop -it lsstts/conda_package_builder``
+Locally test the recipe by running the following command
+
+.. prompt:: bash
+
+    docker run -e {config_repo}=/path/to/config/repo \\
+        -e idl_version="" -e salobj_version="" \\
+        -v /path/to/your/repos:/home/saluser/develop \\
+        -it lsstts/conda_package_builder
+
 Then run the following inside of the container.
 
 .. prompt:: bash
```

Comment:
The last line in this prompt needs to become `conda build --variants "{salobj_version: '', idl_version: ''}' --prefix-length 100 .`. No need for the environment variables in the previous prompt.

Reply:
Thanks, that's very useful. I pushed a fixed version of the file already.